terraform/modules/spack/vpc.tf

```diff
@@ -6,25 +6,33 @@ module "vpc" {
   name = "spack-${var.deployment_name}"
   cidr = var.vpc_cidr
 
-  azs = var.availability_zones
-  public_subnets = var.public_subnets
-  private_subnets = var.private_subnets
-  database_subnets = var.database_subnets
-
-  # Create a DB subnet group for RDS (see rds.tf)
-  create_database_subnet_group = true
```

Comment:
Is `database_subnets` omitted because we don't use the `create_database_subnet_group` var? It seems this defaults to `true`, so wouldn't we want to explicitly set this `false`?

Reply:
Good catch, yes that should be explicitly set to false. I cherry-picked these changes out of a much messier branch and didn't run a plan with them yet, will update this